cms/templates/js/verification-access-editor.underscore

```diff
@@ -0,0 +1,33 @@
+<form>
+<h3 class="modal-section-title"><%= gettext('Verification Checkpoint') %></h3>
+<div class="modal-section-content verification-access">
+  <ul class="list-fields list-input">
+    <li class="field field-checkbox checkbox-cosmetic">
+      <input
+        type="checkbox"
+        id="verification-access-checkbox"
+        class="input input-checkbox"
+        name="verification-access"
+        value=""
+        <% if (hasVerificationRestrictions) { %> checked="checked" <% } %>
+      >
+      <label class="label" for="verification-access-checkbox">
+        <i class="icon fa fa-check-square-o input-checkbox-checked"></i>
+        <i class="icon fa fa-square-o input-checkbox-unchecked"></i>
+
+        <select id="verification-partition-select">
+          <% for (var i = 0; i < verificationPartitions.length; i++) {
+            partition=verificationPartitions[i];
+          %>
+            <option
+              value="<%- partition.id %>"
+              <% if (partition.selected) { %> selected <% } %>
+            ><%- partition.name %></option>
+          <% } %>
+        </select>
+      </label>
+      <span class="note"><%= gettext("Learners who require verification must pass the selected checkpoint to see the content in this unit. Learners who do not require verification see this content by default.") %></span>
```

Comment:
@wedaly That is different than the one you showed me this morning.  I created a gist for that one here:
https://gist.github.com/cptvitamin/bf4b5f9afbfff3af9894

For this, I would follow the same pattern, except the ```<label>``` for the checkbox will have to be SR text as well (both the checkbox and the select will need offscreen labels).

This one also introduces "help" text.  The container for that text needs to have an ID on it so that you can add an ```aria-describedby``` attribute to the checkbox.  A screen reader user will hear that text read when they move focus to the checkbox (in addition to the offscreen label)